docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -37,7 +37,7 @@ The main features of the Image Util API include the following:
 
```

Comment:
This is not MediaVision API. It is Image Util API.
I think it should be fixed by an owner of Image Util module.

Reply:
Will fix it. Thanks. :)